common/utils/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -161,8 +160,17 @@ trait Logging {
 
       MessageWithContext(sb.toString(), context)
     }
+
+    def log(c: Char): MessageWithContext = {
+      MessageWithContext(s"$c", new java.util.HashMap[String, String]())
+    }
   }
 
+  private final val LF_CHAR: Char = '\n'
+  private final val TAB_CHAR: Char = '\t'
+  final val LF = MessageWithContext(s"$LF_CHAR", new java.util.HashMap[String, String]())
```

Comment:
Wait..are you suggesting that we should use this constant in log messages? If so, I would prefer `try...catch` as @HyukjinKwon suggested.

Reply:
This is just to make it `easier` for developers to use commonly used `escape` characters, such as:
line break - `LF_CHAR`, tab - `TAB_CHAR`
Of course, we can also use it this way:
```

```